packages/react-native-worklets/Common/cpp/worklets/SharedItems/Serializable.h

```diff
@@ -362,13 +362,22 @@ class SerializableString : public Serializable {
 
 class SerializableBigInt : public Serializable {
  public:
-  explicit SerializableBigInt(jsi::Runtime &rt, const jsi::BigInt &bigint)
-      : Serializable(BigIntType), string_(bigint.toString(rt).utf8(rt)) {}
+  explicit SerializableBigInt(jsi::Runtime &rt, const jsi::BigInt &bigInt) : Serializable(BigIntType) {
+    if (bigInt.isInt64(rt)) {
+      value_ = bigInt.getInt64(rt);
+    } else {
+      string_ = bigInt.toString(rt).utf8(rt);
+    }
+  }
 
   jsi::Value toJSValue(jsi::Runtime &rt) override;
 
  protected:
-  const std::string string_;
+  /**
+   * This member is used only when the BigInt fits into int64_t range.
+  */
+  std::optional<int64_t> value_{};
```

Comment:
The name `value_` makes sense both for `string` and `int64_t`, how about `number_` or `int64_` instead?
```suggestion
  std::optional<int64_t> int64_{};
```